### ISSUE TITLE
Assembly attributes and the Copyright

### ISSUE THREAD

**Issue by zvirja** [AutoFixture/AutoFixture#750]
Let's discuss the assembly attributes we have. Currently attributes are the following:
```
[assembly: AssemblyCompany("Ploeh")]
[assembly: AssemblyProduct("AutoFixture")]
[assembly: AssemblyCopyright("Copyright © Ploeh 2015")]
```

I'd suggest to change them as following:
```
[assembly: AssemblyCompany("AutoFixture")]
[assembly: AssemblyProduct("AutoFixture")]
```

However it's a question what should be the `Copyright` value. Do we need to change this field or we should keep as is? Or should we to change it to something like `Copyright © Ploeh 2015, AutoFixture 2017`.

@core team, what would you advice? Probably @ploeh can also participate here, as this is his copyright :)

**Comment by ploeh** [comment]
IANAL, but I'm not sure that you can retroactively remove a copyright, although it probably could be sold or transferred. In the end, neither `Ploeh` nor `AutoFixture` are legal entities, so I don't think it matters much.

[Less than a day ago](https://github.com/AutoFixture/AutoFixture/issues/703#issuecomment-292835837), @jongalloway suggested joining the .NET Foundation. If you do that, the good people there might be able to help you sort out issues like this one.

**Comment by zvirja** [comment]
@ploeh I agree that removal of the copyrights is not something straightforward, that's why I started this topic. I had idea to have something combined, like `Copyright © AutoFixture 2017, Ploeh 2011-2017`. I spied that on NUnit:
```csharp
[assembly: AssemblyCopyright("Copyright (C) 2002-2012 Charlie Poole.\r\nCopyright (C) 2002-2004 James W. Newkirk, Michael C. Two, Alexei A. Vorontsov.\r\nCopyright (C) 2000-2002 Philip Craig.\r\nAll Rights Reserved.")]
```

But IANAL as well and I never knew what to write in this field 😉Let's see whether we could count on on somebody else's help.
